### DISCUSSION TITLE
Single letter shortcut based on what what "clicked" last

### DISCUSSION THREAD

**Discussion by alexbenfica** [gdh1995/vimium-c#1205] [Ideas]
Sometimes while doing repetitive tasks, we tend to click the same sequence over and over.

I have to delete 25 user accounts.
They all follow the same pattern: click the first delete button, click confirm. 

The confirm was always in the same position, but the shortcut to it was "SM".
If keeping a track of the screen positions where buttons were clicked last, you could make the single letter shortcuts around the same position, make it even faster to do achieve tasks.

Thank you for your work.

**Comment by gdh1995** [comment]
Um, if there's no confirm button, then you may use `LinkHints.click` with complex parameters to find a target and then click it directly, and even repeat this process.

But now that there is, then you'll have to use:
1. configure `map xxx LinkHints.click match=".css-to-select-the-button"`
2. press `xxx` to trigger it manually after the confirmation dialog shows
3. use `f` to select another account's "delete" button.
4. I suggest you use `<f1> - <f12>` to replace `xxx`, so that you may trigger it "by a single pressing"

Although in theory it's possible to make Vimium C `click A[1] and B, A[2] and B, ..., and then repeat up to 25 times`, the key mapping will be very very complex and it's so hard to debug that I don't think the writing and debugging work worths.

Another replacement is to just write JavaScript code to do so, and then use `map yyy openBookmark title="yyy"` to run it.
